src/server/grpcServer.go

```diff
@@ -303,7 +303,18 @@ func (ps *publisherServer) GetSummary(req *ppb.SummaryRequest, srv ppb.Publisher
 }
 
 func StartGrpcServer() *grpc.Server {
-	s := grpc.NewServer()
+	var s *grpc.Server
+	if viper.GetBool("server.tls.enable") {
+		creds := GetTLSCredentails()
+		if creds != nil {
+			s = grpc.NewServer(grpc.ServerOption(grpc.Creds(creds)))
+		} else {
+			log.Error().Msgf("Unable to read credentails ::  %s", creds)
```

Comment:
```suggestion
			log.Fatal().Msgf("Unable to read credentails ::  %s", creds)
```
It should be `Fatal`. In the program reaches this error state, it cannot proceed after that.

Reply:
addressed